libraries/AP_Notify/Display.h

```diff
@@ -36,5 +37,8 @@ class Display: public NotifyDevice {
     uint8_t _mstartpos;
     uint8_t _movedelay;
     uint8_t _screenpage;
+
+    // stop showing text in display after this many millis:
+    const uint16_t _send_text_valid_millis = 10000;
```

Comment:
@OXINARF san.
The value of _send_text_valid_millis does not change.
If it does not change, I think #define is good.
However, if you want to change the value with debugging or patching, I think that you can leave the variable as it is.